x/erc20/keeper/ibc_callbacks.go

```diff
@@ -85,30 +88,30 @@ func (k Keeper) OnRecvPacket(
 		return ack
 	}
 
+	// If ERC20 module is disabled, dont deploy new precompiles
+	if !k.IsERC20Enabled(ctx) {
+		return ack
+	}
+
```

Comment:
if the ERC-20 module is disabled, can we already exit directly at the top of the function?

Reply:
Moved up the check